src/components/Verification/VerificationButton.jsx

```diff
@@ -0,0 +1,298 @@
+    const app = initializeApp(resourcesConfig);
+    const database = getDatabase(app);
+    // Removed ID since we don't want that as part of the saved data structure
+    const { id, ...filteredResource } = resource;
+    set(ref(database, `/${resource.id}`), filteredResource);
+    setHasBeenUpdated(true);
+    dispatch(updateExistingResource({resource}));
+    dispatch(setSelectedPlace(resource));
+  };
+
+  const markAsVerified = useCallback(() => {
+    const newVerification = {
+      verified: true,
+      last_modifier: name || 'phlask_app',
+      last_modified: new Date().toISOString()
+    };
+    const newResource = {
+      ...resource,
+      verification: newVerification
+    };
+    updateFirebaseEntry(newResource);
+  }, [name, resource.verification]);
+
+  const markAsUnverified = useCallback(() => {
+    const newVerification = {
+      verified: false,
+      last_modifier: name || 'phlask_app',
+      last_modified: new Date().toISOString()
+    };
+    const newResource = {
+      ...resource,
+      verification: newVerification
+    };
+    updateFirebaseEntry(newResource);
+  }, []);
+
+  const markAsInactive = useCallback(() => {
+    const newVerification = {
+      verified: false,
+      last_modifier: name || 'phlask_app',
+      last_modified: new Date().toISOString()
+    };
+    const newResource = {
+      ...resource,
+      status: 'HIDDEN',
+      verification: newVerification
+    };
+    updateFirebaseEntry(newResource);
+  }, []);
+
+  if (!resource) {
+    return;
+  }
+
+  return (
+    <div
+      style={{
+        position: 'absolute',
+        right: '10px',
+        bottom: '10px',
+        backgroundColor: resource.verification.verified ? 'Green' : 'Tomato',
+        padding: '5px',
+        borderRadius: '5px',
+        color: 'white',
+        fontSize: '12px',
+        fontWeight: 'bold',
+        textAlign: 'center'
+      }}
+    >
+      <div onClick={() => setIsModalOpen(true)} style={{ cursor: 'pointer' }}>
+        {resource.verification.verified ? 'VERIFIED' : 'UNVERIFIED'}
+      </div>
+      <Dialog open={isModalOpen} onClose={closeModal}>
+        <div
+          style={{
+            width: '300px',
+            padding: '16px'
+          }}
+        >
+          {!isAdmin && (
+            <div>
+              <p>
+                Enter the admin password to update the verified status of this
+                resource. Also enter your name.
+              </p>
+              <Input
+                placeholder="Your name"
+                style={{ width: '100%', marginBottom: '10px' }}
+                type="text"
+                autoComplete="disabled"
+                value={name}
+                onChange={e => {
+                  setName(e.target.value);
+                }}
+              />
+              <Input
+                placeholder="The admin password"
+                style={{ width: '100%', marginBottom: '10px' }}
```

Comment:
Similar to other mui components, we should use the `sx` attribute here in place of `style`

Reply:
Fixed!